cmd/podman/common/create_opts.go

```diff
@@ -383,6 +384,9 @@ func ContainerCreateToContainerCLIOpts(cc handlers.CreateContainerConfig, rtc *c
 	if cc.HostConfig.Memory > 0 {
 		cliOpts.Memory = strconv.Itoa(int(cc.HostConfig.Memory))
 	}
+	if cc.HostConfig.KernelMemory > 0 {
+		logrus.Warnf("The --kernel-memory flag has been deprecated. May not work properly on your system.")
```

Comment:
Recommend rewording to "May not work properly with some OCI runtimes"

Reply:
Well it is not only the OCI Runtimes, even with crun, it is questionable whether or not the kernel works correctly when this is set in cgroupv2.